src/js/components/Button/stories/Busy.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Box, Button } from 'grommet';
+
+export const Busy = () => {
+  const [busy, setBusy] = useState();
+
+  return (
+    <Box align="center" pad="medium">
+      <Button
+        primary
+        busy={{
+          state: busy,
+          messages: {
```

Comment:
It might be nice for these messages to have a reasonable default string in Button so callers wouldn't have to implement each time

Reply:
I see the default strings now -- you can ignore this comment. Wondering if we should have a story example without custom message overrides to show you don't have to override?

Reply:
And if that's the case can the API for that version be simplified to:

`<Button busy={busy} />` ?

Reply:
I removed `messages` from the story example